Build doesnt work for CMakeList.txt

basically i followed the installation steps given on Installation step on my new virtual machine in Ubuntu 16.04 

I got following error
[ 66%] Completed 'googletest-download'
[100%] Built target googletest-download
CMake Error at build/googletest-src/CMakeLists.txt:13 (if):
  if given arguments:

    "CMAKE_VERSION" "VERSION_GREATER_EQUAL" "3.0.2"

  Unknown arguments specified


Similar issue is created on https://github.com/google/googletest/issues/2711
but it wasnt helpful in my case 
Anyone else facing the same problem ?

Thank you.

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -4,9 +4,9 @@ set(INSTALL_GTEST OFF CACHE BOOL "Install google-test" FORCE)
 
 include(${PROJECT_SOURCE_DIR}/cmake/DownloadProject.cmake)
 download_project(PROJ googletest
-                 GIT_REPOSITORY https://github.com/google/googletest.git
-                 GIT_TAG master
-                 UPDATE_DISCONNECTED 1)
+                 URL https://github.com/google/googletest/archive/release-1.10.0.tar.gz
+                 URL_HASH SHA1=9c89be7df9c5e8cb0bc20b3c4b39bf7e82686770
+                 DOWNLOAD_NO_PROGRESS YES)
 add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR})
 
 # ----- Glob test codes----- 

